src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/SQL/Migrations.cs

```diff
@@ -245,7 +245,7 @@ public int Create()
                 .Column<string>("ContentField", column => column.WithLength(ContentItemIndex.MaxContentFieldSize))
                 .Column<bool>("Published", column => column.Nullable())
                 .Column<bool>("Latest", column => column.Nullable())
-                .Column<DateTime>("Time", column => column.Nullable())
+                .Column<TimeSpan>("Time", column => column.Nullable())
```

Comment:
TimeSpan is only supported by SQL Server we can't use it for SQLite this is why we are using a DateTime mostly.

Reply:
Don't worry, it's converted by the YesSql stack.

Reply:
Ok I take back what I said it seems like @sebastienros added a TimeSpanHandler in YesSQL.